### PR TITLE
style for playground error

### PR DESCRIPTION
**Pull request by mveres**
light
image

dark
image

### CODE REVIEW

File: playground/src/Playground.tsx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import dracula from 'prism-react-renderer/themes/dracula';
 import toJSXString from 'react-element-to-jsx-string';
 
 const styles = {
   preview: {
     padding: '1rem',
-    border: '1px solid #e1e4e8',
-  },
+    border: '1px solid #E1E4E8',
```
Comment (gqio) [inline playground/src/Playground.tsx:9]: It's a bit too much I think on dark mode, I would go for something like `hsla(0, 0%, 50%, .2)` , normally it's working on dark/light, WDYT ?

File: playground/src/Playground.tsx [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+  } as CSSProperties,
+  editor: {
+    height: 'auto',
+    overflow: 'auto',
+  } as CSSProperties,
+  error: {
+    display: 'block',
```
Comment (gqio) [inline playground/src/Playground.tsx:16]: I think it's a `<pre>` and it's already a `block`

File: playground/src/Playground.tsx [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 const styles = {
   preview: {
     padding: '1rem',
-    border: '1px solid #e1e4e8',
-  },
+    border: '1px solid #E1E4E8',
+  } as CSSProperties,
+  editor: {
+    height: 'auto',
+    overflow: 'auto',
```
Comment (gqio) [inline playground/src/Playground.tsx:13]: I think it's not mandatory here, since there is no max-height or fixed height, I didn't checked.

**Comment by mveres** [comment]
updated screenshots

**Review by gqio — APPROVED** [review]

**Pull request merged**